waypoints-special/CAN_PG_A-M.cup

```diff
@@ -1,415 +1,414 @@
 "Two Hills",TwoHills,CA,5342.000N,11145.964W,596.0m,1,,,,
-"Vankleek Hill",VNKLKHLL,CA,4531.183N,07439.133W,115.8m,1,,,,"Turn Point, Rd. Int."
-"Vegreville",Vegrevil,CA,5330.867N,11201.650W,631.5m,5,130,1219.2m,"122.8","Turnpoint, (UNICOM o/t tfc).  Righthand circuits Rwy 31.  NOTE:  Gun club 1 nm NW.  Avoid or overfly no less than 1000 ft AGL.  Ultralight activity in vicinity.  13/31."
+"Vankleek Hill",VNKLKHLL,CA,4531.183N,07439.133W,115.8m,1,,,,
+"Vegreville",Vegrevil,CA,5330.867N,11201.650W,631.5m,5,130,1219.2m,"122.8",(UNICOM o/t tfc).  Righthand circuits Rwy 31.  NOTE:  Gun club 1 nm NW.  Avoid or overfly no less than 1000 ft AGL.  Ultralight activity in vicinity.  13/31."
 "Wimborne Hamlet",WINBOH,CA,5151.600N,11335.500W,946.0m,1,,,,
 "Woodstock",Woodsown,CA,4306.582N,08049.248W,295.7m,1,,,,
 -----Related Tasks-----
-"AVVC 1 - St Dominique","CSS4","St Dominique City","St Liboire","Ste Rosalie","St Dominique City","CSS4"
+"AVVC 1 - St Dominique","CSS4","St Dominique City","St Liboire","Ste Rosalie","St Dominique City","CSS4",,,,
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
-ObsZone=0,Style=2,R1=2000m,A1=180
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
-ObsZone=3,Style=3,R1=2000m,A1=180
+ObsZone=3,Style=3,R1=2000m,A1=180,Move=1
 "AVVC 1 - St Liboire","CSS4","St Liboire","Ste Rosalie","St Dominique City","St Liboire","CSS4"
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
-ObsZone=0,Style=2,R1=2000m,A1=180
+ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
-ObsZone=3,Style=3,R1=2000m,A1=180
+ObsZone=3,Style=3,R1=2000m,A1=180,Move=1
 "AVVC 1 - Ste Rosalie","CSS4","Ste Rosalie","St Dominique City","St Liboire","Ste Rosalie","CSS4"
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
-ObsZone=0,Style=2,R1=2000m,A1=180
+ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
-ObsZone=3,Style=3,R1=2000m,A1=180
+ObsZone=3,Style=3,R1=2000m,A1=180,Move=1
 "AVVC 1inv - St Dominique","CSS4","St Dominique City","Ste Rosalie","St Liboire","St Dominique City","CSS4"
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
-ObsZone=0,Style=2,R1=2000m,A1=180
+ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
-ObsZone=3,Style=3,R1=2000m,A1=180
+ObsZone=3,Style=3,R1=2000m,A1=180,Move=1
 "AVVC 1inv - St Liboire","CSS4","St Liboire","St Dominique City","Ste Rosalie","St Liboire","CSS4"
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
-ObsZone=0,Style=2,R1=2000m,A1=180
+ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
-ObsZone=3,Style=3,R1=2000m,A1=180
-"AVVC 1inv - Ste Rosalie","CSS4","Ste Rosalie","St Liboire","St Dominique City","Ste Rosalie","CSS4"
+ObsZone=3,Style=3,R1=2000m,A1=180,Move=1
+"AVVC 1inv - Ste Rosalie","CSS4","Ste Rosalie","St Liboire","St Dominique City","Ste Rosalie","CSS4",,,,
```

Comment:
The format of this line is "Description","wp1","wp2" the commas at the end are not necessary and might confuse the parser.

Reply:
Thank you.  I was switching to a comma delimited file and these were remnants.  I opened the file in SeeYou again and then saved.  That got rid of it.